packages/core/README.md

```diff
@@ -220,16 +230,66 @@ A full documentation of the masterportalapiPolygonFillHatch is available at the
 >|patternColor|no|Number[]|`[255, 255, 255, 1]`|Fill color of pattern drawn on polygon.|
 >|size|no|Number|`30`|Edge length of a singular repeated pattern element.|
 
-##### mapConfiguration.LayerConf
+##### <...masterportalapi.fields>
+
+The `<...masterportalapi.fields>` means that any masterportalapi field may also be used here _directly_ in the mapConfiguration. The fields described here are fields that are interesting for the usage of POLAR.
+Fields that are not set as required have default values.
+
+| fieldName | type | description |
+| - | - | - |
+| layerConf | layerConf | Layer configuration of all available layers as a service register. Layers defined here are not directly shown in a client, see `mapconfiguration.layers` for that. |
+| layers | layer[] | Configuration of layers that are supposed to be used in the respective client. All layers defined here have to have an entry in `mapConfiguration.layerConf`. If `@polar/plugin-layer-chooser` is installed and configured, all these layers will be displayed in that menu. |
+| startCenter | number[] | Initial center coordinate. Needs to be defined in the chosen leading coordinate system. |
+| backgroundImage | string? | Image to be displayed as a background of the map. Defaults to `''`, so no image being used. |
+| epsg | `EPSG:${string}`? | Leading coordinate system. The coordinate system has to be defined in `mapConfiguration.namedProjections` as well. Changing this value should also lead to changes in `mapConfiguration.startCenter`, `mapConfiguration.extent`, `mapConfiguration.options` and `mapConfiguration.startResolution` as they are described in or related to the leading coordinate system. Defaults to `'EPSG:25832'`. |
+| extent | number[]? | Map movement will be restricted to the rectangle described by the given coordinates. Unrestricted by default. |
+| namedProjections | Array<[string,string]>? | Array of usable coordinated systems mapped to a projection as a proj4 string. Defines the `'EPSG:25832'`, `'EPSG:3857'` and `'EPSG:4326'` by default. |
```

Comment:
```suggestion
| namedProjections | Array<[string,string]>? | Array of usable coordinated systems mapped to a projection as a proj4 string. Defines the `'EPSG:25832'`, `'EPSG:3857'` and `'EPSG:4326'` by default. If you set a value, please mind that all pre-configured projections are overridden, and requiring e.g. `'EPSG:4326'` will only work if it is also defined in your override. |
```

I see this going wrong the first time someone uses this, so I'd add this extra cautious sentence to make sure people don't run into hard-to-debug problems.